Update windows-local-autopilot-reset.md (#3470)
MicrosoftDocs/windows-itpro-docs: windows/deployment/windows-autopilot/windows-autopilot-reset-local.md

**Applies to: Windows 10, version 1709 and above

> The Microsoft Intune Service Administrator Directory role is required to perform this task. The process of adding directory roles is documented at https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-users-assign-role-azure-portal

Reviewer: ```suggestion
The Microsoft Intune Service Administrator Directory role is required to perform this task. To learn more about the process of adding directory roles, refer to https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-users-assign-role-azure-portal
```

> The Microsoft Intune Service Administrator Directory role is required to perform this task. The process of adding directory roles is documented at https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-users-assign-role-azure-portal

Reviewer 2: ```suggestion
The Intune Service Administrator role is required to perform this task.  Learn more about how to [Assign Azure Active Directory roles](https://docs.microsoft.com/azure/active-directory/fundamentals/active-directory-users-assign-role-azure-portal).
```

> The Microsoft Intune Service Administrator Directory role is required to perform this task. To learn more about the process of adding directory roles, refer to https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-users-assign-role-azure-portal

Reviewer 3: ```suggestion
The Intune Service Administrator role is required to perform this task.  Learn more about how to [Assign Azure Active Directory roles](https://docs.microsoft.com/azure/active-directory/fundamentals/active-directory-users-assign-role-azure-portal).
```
